### DISCUSSION TITLE
Why is fingerprinting privacy friendlier then cookies?

### DISCUSSION THREAD

**Discussion by ihucos** [plausible/analytics#2403] [Anything Else]
Full disclosure: I maintain my [own little web analytics platform](https://github.com/ihucos/counter.dev). So you might take that question with a little grain of salt and feel free to ask questions back I guess.

First of all actually I think plausible is a great product by the way and can praise the extensive documentation around it. To the question: almost all privacy friendly analytics product use fingerprinting techniques rather than cookies. I understand this as instead of giving end-users cookies with an identifier, the needed identifier is generated from the users IP address, user agent and possibly other data.

When cookies are used, as an end user I can know that I am being "tracked" and can delete my cookies so I am "forgotten" or don't accept cookies at all in order to not being tracked. With fingerprinting techniques, as an end user I don't have that control.

So my purely technical question is how fingerprinting is more privacy friendly than cookies.

Thank you in advance!

**Comment by ukutaht** [comment]
Politics. I don't see anything wrong with a first-party cookie. The reason most privacy-focused platforms don't use cookies is due to the ePrivacy directive which requires a cookie banner if they're used for analytics purposes. This may be change with the upcoming ePrivacy regulation (as opposed to directive) but I'm not read up on it currently: https://www.lexology.com/library/detail.aspx?g=2c0eca0b-c828-4fd6-ac0f-21fdbeded2bb

**Reply by ihucos** [reply]
Thanks for the response. Yes, I don't see any technical founded reasoning for preferring fingerprinting over cookies for more privacy friendly web analytics. Ultimately in this case the politics lead to less privacy friendly products.

**Reply by ukutaht** [reply]
Agreed
